nltk/lm/models.py

```diff
@@ -92,8 +119,23 @@ def __init__(self, order, **kwargs):
         super().__init__(WittenBell, order, **kwargs)
 
 
+class AbsoluteDiscountingInterpolated(InterpolatedLanguageModel):
+    """Interpolated version of smoothing with absolute discount."""
+
+    def __init__(self, order, discount=0.75, **kwargs):
+        super().__init__(
+            AbsoluteDiscounting, order, params={"discount": discount}, **kwargs
+        )
+
+
 class KneserNeyInterpolated(InterpolatedLanguageModel):
     """Interpolated version of Kneser-Ney smoothing."""
 
     def __init__(self, order, discount=0.1, **kwargs):
```

Comment:
@manujosephv I undid your change of default value here. I think we add enough entropy for existing users by changing the scores, let's not throw this in as well. I've set the value to `0.75` in the tests, so they didn't need to be changed.